### DISCUSSION TITLE
How to Use Org to control Custom Model?

### DISCUSSION THREAD

**Discussion by wbond** [github-early-access/copilot-custom-models-beta#5] [Q&A]
Currently we are gating rollout of the custom model by having a custom org that we assign users to.

While working to try and perform some really basic evaluation before adding more users, one of our members tried to remove themselves from GitHub Copilot Access in the org settings.

As a result, their name was greyed out and the interface now says they'll be removed at the end of the billing period.

![CleanShot 2024-09-19 at 14 06 43@2x](https://github.com/user-attachments/assets/6d3c1c8b-0f90-4ed5-b94f-d0a1357df2b6)

Do you have suggestions for how to move people in and out of access? Will we need to remove them from the org altogether to get the custom model to stop being used for their completions?

**Comment by thejoebourneidentity** [comment]
I believe you'll just need to remove their Copilot seat under the Copilot->Access settings. I don't think they need to be totally removed from the org. cc @ivanpk is this accurate?

**Comment by wbond** [comment]
The issue with removing their access is as shown in the screenshot above - the access stays until the end of the billing cycle.

**Reply by thejoebourneidentity** [reply]
Ah, I see. @justincampbell this seems to be a detail in how the licenses are managed once a seat is granted and later removed. Are you able to gleam from how the access works whether or not the user needs to be completely removed from the org?

**Reply by thejoebourneidentity** [reply]
Hey @wbond, @ivanpk just looked into this and found that if you need an immediate fix you should need to remove the user from the organization as well, otherwise they'll remain until the end of the billing cycle.
